build/pr-variables.yml

```diff
@@ -3,7 +3,8 @@ variables:
     resourceGroupRoot: 'msh-fhir-pr'
     appServicePlanName: '$(resourceGroupRoot)-$(prNumber)-asp'
     prNumber: $(system.pullRequest.pullRequestNumber)
-    DeploymentEnvironmentName: '$(resourceGroupRoot)-$(prNumber)'
+    ResourceGroupName: '$(resourceGroupRoot)-$(prNumber)'
+    DeploymentEnvironmentName: 'f$(build.BuildNumber)'
```

Comment:
>f [](start = 32, length = 1)

why plain `f` is better than `fhir`?

Reply:
It is shorter. I'm running into issues with the full build number being too long for some resource types. There is a max name length of 24 characters.